### ISSUE TITLE
cDNA reads statistics bug?

### ISSUE THREAD

**Issue by liufuyan2016** [MGI-tech-bioinformatics/DNBelab_C_Series_HT_scRNA-analysis-software#126]
We observed that in the **metrics_summary.xls** file under **04.report**, the cDNA Number of reads is **1,858,750,737**, but in the **anno_decon_sorted.bam** file under the **output** directory, there are **2,172,605,62**7, significantly more than the final statistical result. I am not sure if there is a bug in your statistics, **which is quite confusing.**

**Comment by lishuangshuang0616** [comment]
multi mapping and bam file remove the unmapping reads.

**Comment by liufuyan2016** [comment]
You say in the final metrics_summary.xls  file， the statistics is removed multi mapping and unmapping reads?

**Comment by liufuyan2016** [comment]
The cDNA Number of reads I sugguest use the unique mapping  cDNA Number of reads?

**Comment by lishuangshuang0616** [comment]
cDNA Number of reads is the raw fastq reads of cDNA library.

**Comment by liufuyan2016** [comment]
> multi mapping and bam file remove the unmapping reads.

I make statistics use **samtools flagstat** , The results read number is higher than the raw fastq reads of cDNA library in the metrics_summary.xls file

**Comment by lishuangshuang0616** [comment]
Yes, because there are many multiple alignments. You can open it and see that the MAPQ values ​​that are not 255 are all sequences with multiple alignments.

**Comment by liufuyan2016** [comment]
Thank you! I known the samtools flagstat  could not distinguish the multi mapping.

**Comment by liufuyan2016** [comment]
I also want to ask if multiple aligned reads are used for cell read counting? for example Mean reads per cell

**Comment by lishuangshuang0616** [comment]
![image](https://github.com/user-attachments/assets/54733ecf-aed7-4c65-aad2-9d78026aa004)
For reads with multiple alignments, we will filter the results on the alignment and only select one of them, and we will select it based on the alignment area.
